### ISSUE TITLE
Training time cost

### ISSUE THREAD

**Issue by Liwen-Xiao** [opendilab/SmartRefine#12]
Hello! Great job! Could you tell me how long your training time is? On my machine, using a single 3090 GPU, the first epoch shows it will take around 8 hours to train. Is this similar to your experience?

**Comment by youngzhou1999** [comment]
Hi. Thanks for your interest.
The first epoch is relatively longer as it will process data for training, you can reduce the radius of map retrieval to fasten your training process.

**Comment by Liwen-Xiao** [comment]
Thank you for your reply! I changed the radius and the training cost decreased a lot. Thanks again!

**Comment by lon0862** [comment]
Hi, but I find my training cost is take around 8 hours in each epoch, not only the first epoch.  Is this similar to your experience?

**Comment by Liwen-Xiao** [comment]
> Hi, but I find my training cost is take around 8 hours in each epoch, not only the first epoch. Is this similar to your experience?

Yes, it is the same. I changed the local radius to a smaller one and solved the problem.

**Comment by lon0862** [comment]
> Yes, it is the same. I changed the local radius to a smaller one and solved the problem.
> 
Thanks, I do the same thing to solved it, too.

**Comment by lon0862** [comment]
@Liwen-Xiao  Hi, I want to ask about the retrained result. I use 1 GPU, batch_size: 32, accumulate by 2 batches。Final only get val_minADE: 0.6525, val_minFDE: 0.9325, val_MR: 0.0880。And in preprocess data, I use local_radius:65。
I want to know have you get the result same as github checkpoints?

**Comment by Liwen-Xiao** [comment]
Hi, I also retrained the model. I got val_minFDE as 0.919, which is close to 0.913 as the author reported. I use 1 GPU, batch size is 32, and the epoch is 64. I get the best result at the 33rd epoch.

**Comment by lon0862** [comment]
Hi, I use the same parameter as you show. And I get the best result at the 31st epoch. which val_minADE is 0.649, 
val_minFDE:0.924, val_MR:0.086. It is a little worse than yours and author reported, but I think it is acceptable. Thanks!

**Comment by lon0862** [comment]
But I still confuse about the quality score using, do you try it as paper says?

**Comment by weekiiy** [comment]
Do you have the same situation, I trained more than 40 epochs and the loss can not be reduced to around 8, how to solve this?

**Comment by Yurriki** [comment]
@Liwen-Xiao Sorry to bother you, but I would like to ask how long it takes to train for about 40 rounds on a single 3090 graphics card with the same experimental setup as yours
